processing/src/main/java/io/druid/segment/IndexIO.java

```diff
@@ -91,12 +93,13 @@
   private static final SerializerUtils serializerUtils = new SerializerUtils();
 
   private final ObjectMapper mapper;
+  private final ColumnConfig columnConfig;
```

Comment:
I'm not sure it's the correct point to move ColumnConfig out of IndexIO in this PR, because there are so many test code need be modified and they are irrelevant to this PR.
I think it's better do the refactor in another PR